Source/Core/Common/Logging/Log.h

```diff
@@ -33,6 +33,7 @@ enum class LogType : int
   FRAMEDUMP,
   GDB_STUB,
   GPFIFO,
+  HSP,
   HOST_GPU,
```

Comment:
These aren't alphabetized properly (the order in LogManager.cpp is correct though).